src/DatesSimple.java

```diff
@@ -0,0 +1,44 @@
+public class DatesSimple {
+    public static void main(String[] args) {
+        int fromy = 2000, fromm = 2, fromd = 3;
+        int toy = 2127, tom = 11, tod = 29;
+
+        System.out.println("**** Original\n" +
+                "From " + fromy + "/" +
+                fromm + "/" + fromd + " to " +
+                toy + "/" + tom + "/" + tod);
+//      pack
+        int period = 0;
+        int resetFromy = fromy - 2000;
+        int resetToy = toy - 2000;
+
```

Comment:
Please add a comment explaining what your solution does

Reply:
Please add a comment = add a comment in your code, where will you explain your solution.

Reply:
![image](https://github.com/user-attachments/assets/1f04c46b-fbcb-4cd0-9697-4b3880aa2ce1)